### PR TITLE
:memo:Translating docs to Simplified Chinese

### PR DESCRIPTION
**Pull request by miaowumiaomiaowu**
#1915 
I have translated "search.md" into Simplified Chinese. 😊

### CODE REVIEW

**Review by stevhliu — COMMENTED** [review]
LGTM, thanks for your contribution! Do you know another contributor who would be interested in reviewing your translation?

**Comment by miaowumiaomiaowu** [comment]
@stevhliu Thanks for the feedback! I believe @13579606 would be a great person to review this translation. @13579606, would you be able to help with reviewing?

File: docs/source/cn/guides/search.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+>>> models = api.list_models()
+```
+
+[`list_models`] 返回一个迭代器，包含存储在 Hub 上的模型。
+
+同样，您可以使用 [`list_datasets`] 列出数据集，使用 [`list_spaces`] 列出 Spaces。
+
+## 如何过滤仓库？
+
+直接列出仓库虽然有用，但您可能想要过滤您的搜索。
```
Comment (13579606) [inline docs/source/cn/guides/search.md:24]: > 直接列出仓库虽然有用，但您可能想要过滤您的搜索。

列出仓库是一个好开始，但现在您可能希望对搜索结果进行过滤。

File: docs/source/cn/guides/search.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+```py
+models = hf_api.list_models(
+	task="image-classification",
+	library="pytorch",
+	trained_dataset="imagenet",
+)
+```
+
+在过滤时，您还可以对模型进行排序并仅获取前几个结果。例如，以下示例获取 Hub 上下载次数最多的前 5 个数据集：
```
Comment (13579606) [inline docs/source/cn/guides/search.md:41]: 在过滤时，您还可以对模型进行排序，并仅获取前几个结果。例如，以下示例获取了 Hub 上下载量最多的前 5 个数据集：

**Review by 13579606 — APPROVED** [review]
Hi, @miaowumiaomiaowu .Here are some suggestions for the translation, don't hesitate to drop me a message if needed.

**Comment by miaowumiaomiaowu** [comment]
Hi @13579606 ,
Thank you so much for your thorough review and valuable suggestions! I really appreciate you pointing out the issues, and I will immediately update and refine the translation file based on your feedback to ensure it is more accurate and clear.
If you have any further suggestions or comments, feel free to let me know.
Thanks again for your help!

**Review by 13579606 — APPROVED** [review]
@miaowumiaomiaowu  Great work! Thanks for the contribution. It looks much better.

**Review by hanouticelina — APPROVED** [review]
Hi @miaowumiaomiaowu, thanks a lot for this contribution 🤗 looks good to me, we are good to merge!

**Comment by hanouticelina** [comment]
And of course thanks @13579606 for the review!

**Pull request merged**